File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F18.P2.java
class Main {
    static int callUtil() {
        // BEGIN PROBE F18.P2
        int result = util.Util.pubAdd(2, 3);
        return result;
        // END PROBE F18.P2
    }

    public static void main(String[] args) {
        System.out.println(callUtil());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F18.P2.cpp
#include <cstdint>
#include <cstdio>

// BEGIN PROBE F18.P2
#include "util.h"

int main() {
    std::int32_t result = util::pub_add(2, 3);
// END PROBE F18.P2
    std::printf("%d\n", result);
}
